privacy_evaluator/metrics/privacy_risk_score.py

```diff
@@ -0,0 +1,75 @@
+from ..classifiers.classifier import Classifier
+import numpy as np
+from typing import Tuple
+
+
+def compute_privacy_risk_score(
+    target: Classifier,
+    x_train: np.ndarray,
+    y_train: np.ndarray,
+    x_test: np.ndarray,
+    y_test: np.ndarray,
+    num_bins=15,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """
+    Computes each individual point's likelihood of being a member
+    (denoted as privacy risk score in https://arxiv.org/abs/2003.10595).
+
+    For an individual sample, its privacy risk score is computed as the posterior
+    probability of being in the training set
+    after observing its prediction output by the target machine learning model.
+
+    (Helper method and description taken from
+    https://github.com/tensorflow/privacy/blob/master/tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py#L217)
+
+
+    :param attack_input: input data for compute membership probability
+    :param num_bins: the number of bins used to compute the training/test histogram
+    :return: membership probability results
+    """
+    loss_train = target.art_classifier.compute_loss(x_train, y_train)
+    loss_test = target.art_classifier.compute_loss(x_test, y_test)
+    return _compute_membership_probability(loss_train, loss_test, num_bins)
+
+
+def _compute_membership_probability(
```

Comment:
can you explain why num_bins is set to 15? Please document.

Reply:
@blauertee can you say something about this?

Reply:
@fraboeni This is just the default value, it's not hard coded. It was the value they used as default in tf privacy. If you don't think a default value makes any sense here I can just make it a required argument.